alcs-frontend/src/app/features/board/recon-create-card-dialog/recon-create-card-dialog.component.html

```diff
@@ -101,14 +101,14 @@ <h2 class="card-title">Create Reconsideration</h2>
       <div>
         <ng-select
           appearance="outline"
-          class="card-region"
-          [items]="regions"
+          class="card-recon-type"
```

Comment:
Why this changed?

Reply:
The order of the fields was not the same as create-application, so I have re-ordered the fields here to put local government to come before region.